go/tasks/plugins/array/k8s/monitor.go

```diff
@@ -192,11 +194,23 @@ func CheckPodStatus(ctx context.Context, client core.KubeClient, name k8sTypes.N
 	}
 
 	if pod.Status.Phase != v1.PodPending && pod.Status.Phase != v1.PodUnknown {
-		taskLogs, err := logs.GetLogsForContainerInPod(ctx, pod, 0, " (User)")
+		p, err := logs.InitializeLogPlugins(&GetConfig().LogConfig.Config)
```

Comment:
why initialize for all calle?

Reply:
Looks like we do the same for spark plugin after the recent logging re-write . I will update this to be less frequent :)